app/src/main/res/values/donottranslate.xml

```diff
@@ -115,7 +115,7 @@
         <item>gab.ai</item>
     </string-array>
 
-    <string name="rick_roll_url">https://www.youtube.com/watch?v=dQw4w9WgXcQ</string>
+    <string name="rick_roll_url">https://www.youtube.com/watch?v=FXPKJUE86d0</string>
```

Comment:
seems to be changing the url without changing the key, might be a bad idea as this string is pretty much being misrepresented and a more appropriate name could be `banned_url` or something